prepare-build.sh

```diff
@@ -67,7 +67,27 @@ else
     if [ "$#" -gt 0 ]; then
         export MACHINE=${1}
     else
-        export MACHINE=dory
+        echo "***** No target defined *****\n"
+        echo "Usage:"
+        echo "$ . ./prepare-build.sh [update | device-name]\n"
+        echo "Available devices:\n"
+
+        echo "anthias"
+        echo "bass"
+        echo "dory"
+        echo "harmony"
+        echo "inharmony"
+        echo "lenok"
+        echo "mooneye"
+        echo "smelt"
+        echo "sparrow"
+        echo "sprat"
+        echo "sturgeon"
+        echo "swift"
+        echo "tetra"
+        echo "wren"
```

Comment:
If we are adding a list of supported devices, we might as well store it into a global array and use it to verify that the user uses a correct ./prepare-build.sh MACHINE